### ISSUE TITLE
[FEATURE] Pre-commit hook support

### ISSUE THREAD

**Issue by arichtman** [sonatype-nexus-community/jake#84]
* [Pre-commit](https://pre-commit.com/) is a popular Python tool for managing, using, and sharing Git hooks. Presently there appear to be no shared hooks for Jake.

* This supports using Jake for local feedback in an automated fashion

* The impact of not implementing is higher barriers to uptake of Jake, as well as some minor duplication of work with developers implementing something locally

* PR incoming. No further comment.

cc @bhamail / @DarthHater

**Comment by arichtman** [comment]
This can be tested prior to merge with the following `.pre-commit-config.yaml`

```Yaml
  - repo: https://github.com/arichtman/jake
    rev: "ft-pre-commit-hooks"
    hooks:
      - id: scan
 ```

**Comment by madpah** [comment]
PR #85 was accepted and we've released `jake` `1.3.0`. Thanks again @arichtman!
